docs/deployment/README.md

```diff
@@ -54,9 +53,9 @@ The process involves:
 
 ## Installation
 
-#### Prepare DAAC deployment repository
+### Prepare DAAC deployment repository
 
-_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration. ](#prepare-aws-configuration) _
+_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration. ](#prepare-aws-configuration)_
```

Comment:
Is this supposed to create italicized text? Because it doesn't seem like it's working on the documentation site:

https://nasa.github.io/cumulus/docs/deployment/deployment-readme#prepare-daac-deployment-repository

Reply:
Yep I fixed it :)